src/bundle/Resources/public/img/icons/order-history.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_3501_639298)">
```

Comment:
maybe we should remove these `<g>` etc. so that we have exactly the same code here as in `ibexa-icons.svg`?